src/org/omegat/core/data/ExternalTMX.java

```diff
@@ -43,9 +43,9 @@
 
     private final String name;
 
-    private final List<PrepareTMXEntry> entries;
+    private final List<ITranslationEntry> entries;
 
-    ExternalTMX(String name, List<PrepareTMXEntry> entries) {
+    ExternalTMX(String name, List<ITranslationEntry> entries) {
```

Comment:
This can be break plugins when plugin use `ExternalTMX` constructor. Any plugin use it?

Reply:
Actually I don't think that OmegaT plugins are using these class. For DGT it is different because I also have translation memory plugins.
That said, to be more generic you can write
ExternalTMX(String name, Iterable<? extends ITranslationEntry> entries) {
then, callers based on List<PrepareTMXEntry> should continue to work.